attributions.html

```diff
@@ -0,0 +1,138 @@
+                        <button class="dropbtn">OUTREACH</button>
+                        <div class="dropdown-content outreach-dropdown-content">
+                            <a href="#">COLLABORATIONS</a>
+                            <a href="#">PUBLIC ENGAGEMENT</a>
+                        </div>
+                    </div>
+                    <div class="dropdown">
+                        <button class="dropbtn">HUMAN-CENTERED DESIGN</button>
+                        <div class="dropdown-content human-dropdown-content">
+                            <a href="#">PRACTICES</a>
+                            <a href="#">POLICIES</a>
+                            <a href="http://2018.igem.org/Team:Cornell/Entrepreneurship">ENTREPRENEURSHIP</a>
+                        </div>
+                    </div>
+                    <div class="dropdown">
+                        <button class="dropbtn">MODELING</button>
+                        <div class="dropdown-content modeling-dropdown-content">
+                            <a href="http://2018.igem.org/Team:Cornell/Model">MODEL</a>
+                            <a href="http://2018.igem.org/Team:Cornell/Software">SOFTWARE</a>
+                            <a href="http://2018.igem.org/Team:Cornell/Design">DESIGN</a>
+                        </div>
+                    </div>
+                    <div class="dropdown">
+                        <button class="dropbtn">TOOLKIT</button>
+                        <div class="dropdown-content toolkit-dropdown-content">
+                            <ul>
+                                <div class="nav-first-col">
+									<li class = "wet-lab-list-title"><b>WET LAB</b></li>
+									<li><a href="http://2018.igem.org/Team:Cornell/Foundations">FOUNDATIONS</a></li>
+									<li><a href="http://2018.igem.org/Team:Cornell/Demonstrate">DEMONSTRATE</a></li>
+									<li><a href="http://2018.igem.org/Team:Cornell/InterLab">INTERLAB</a></li>
+									<li><a href="hhttp://2018.igem.org/Team:Cornell/Parts">PARTS</a></li>
+                                    
+                                </div>
+
+                                <div class="nav-second-col">
+									<li class = "doc-list-title"><b>DOCUMENTATION</b></li>
+									<li><a href="http://2018.igem.org/Team:Cornell/Notebook">NOTEBOOK</a></li>
+									<li><a href="http://2018.igem.org/Team:Cornell/Safety">SAFETY</a></li>
+                                    
+                                </div>
+                            </ul>
+                        </div>
+                    </div>
+                    <div class="dropdown">
+                        <button class="dropbtn" onclick="location.href='http://2018.igem.org/Team:Cornell/Description '">ABOUT</button>
+                    </div>
+                    <div class="dropdown">
+                        <button class="dropbtn active-page" onclick="location.href='http://2018.igem.org/Team:Cornell'">HOME</button>
```

Comment:
Remove "active-page" from the class. Add it to the class for `TEAM` instead. I added a comment for where to add that.